integration/src/test/scala/ammonite/integration/StandaloneTests.scala

```diff
@@ -63,6 +63,31 @@ object StandaloneTests extends TestSuite{
       println("\n-------------------------")
     }
 
+    'error_test{
+      val name = "Error_line_number_test.scala"
+      try {
+        val res = %%bash(
+          executable,
+          "--predef-file",
+          emptyPrefdef,
+          replStandaloneResources / name
+          )
+      }
+      catch
+        {      // expected_error_msg is expected error message for test file
+          case ex: Throwable => val expected_error_msg = """ammonite.ops.ShelloutException:
+                                                           |CommandResult 1##
+                                                           |Exception in thread "main" ammonite&&
+                                                           |.repl.CompilationError: ("}" | `case`)&&
+                                                           |:5:24 ...")\n  }\n\n  d"""".stripMargin.replaceAll("\n"," ")
+                                                           .replaceAll("## ","\n").replaceAll("&& ","")
```

Comment:
If you format the code as described above you won't need all these we're `replaceAll` calls